src/platform_impl/macos/app_state.rs

```diff
@@ -280,7 +280,16 @@ impl AppState {
     }
   }
 
+  pub fn will_launch(app_delegate: &Object) {
+    apply_activation_policy(app_delegate);
+  }
+
   pub fn launched(app_delegate: &Object) {
+    // In catalina, if we set the activation policy before the app is launched,
+    // We get a bug where the menu is not clickable.
+    // We have solved this problem by setting the activation policy to prohibited
+    // with `set_policy_to_prohibited` and then running `apply_activation_policy` again. 
+    set_policy_to_prohibited();
     apply_activation_policy(app_delegate);
```

Comment:
@amrbashir @keiya01 
I checked the operation in catalina and the menu bar was not clickable in the previous commit.
It is not a fundamental solution, but once I set the activation policy to prohibit and then run apply_activation_policy again, I can click on the menu bar.

https://user-images.githubusercontent.com/65934663/175801984-01a24ee6-1488-40a8-9946-7d98694a0b20.mp4

Reply:
I will leave this for @wusyong to approve. He is the macOS expert.

Reply:
Sorry, I totally forgot this issue :(
I'm on Ventura now and this PR couldn't solve it.